src/wasm.h

```diff
@@ -1158,6 +1158,17 @@ struct Importable {
   bool imported() { return module.is(); }
 };
 
+class Function;
+
+// Represents a mapping of wasm module elements to their location in the
+// binary representation. This is used for general debugging info support.
+// Offsets are relative to the beginning of the code section, as in DWARF.
+struct BinaryLocations {
+  using Span = std::pair<uint32_t, uint32_t>;
```

Comment:
Maybe mention what these fields are for? (i.e. the second one could be interpreted as another binary offset or a size).
Come to think of it, should the first be `size_t` instead of uint32_t?
(also this could be avoided if we just used a struct with named elements. it wouldn't even be any more code!)

Reply:
A nice thing of pair is that we don't need to write boilerplate for the constructor etc. And the order is clear. I'll add a comment to clarify, I think that's enough?

Reply:
Fair enough, updated to use a named struct.